src/main/resources/bamm/entity/2.0.0/TimeSeriesEntity.ttl

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2021 Robert Bosch Manufacturing Solutions GmbH
+#
+#  See the AUTHORS file(s) distributed with this work for additional
+#  information regarding authorship.
+#
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+
+@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:2.0.0#> .
+@prefix bamm-c: <urn:bamm:io.openmanufacturing:characteristic:2.0.0#> .
+@prefix bamm-e: <urn:bamm:io.openmanufacturing:entity:2.0.0#> .
+
+bamm-e:TimeSeriesEntity a bamm:AbstractEntity ;
+   bamm:name "TimeSeriesEntity" ;
+   bamm:preferredName "Time Series Entity"@en ;
+   bamm:description "An Entity which represents a key/value pair. The key is the timestamp when the value was recorded and the value is the value which was recorded."@en ;
```

Comment:
I think adding a link to the corresponding specification section is helpful, but I would actually not be happy with adding this link using the Github Pages domain main. When the specification is ever hosted somewhere else, this would break. Ideally, there should be a redirect hosted on open-manufacturing.org or openmanufacturing.io.

Reply:
I have added the link using the GitHub Pages domain for now. Once we have a different domain, the link should be updated.